scripts/publish/prepack.sh

```diff
@@ -25,8 +25,8 @@ rsync -av --include '*/*.sol' --exclude 'test' --exclude 'build' --exclude  'REA
 rm build/src/abi/contracts/*.dbg.json
 rm build/src/abi/interfaces/*.dbg.json
 echo "Start abi export and package.json generation"
-npx ts-node scripts/publish/abi-export.ts
```

Comment:
is there a reason we made this switch? from a brief search it looks like tsx, while faster, does not chekc types

Reply:
ts-node keeps breaking for me, there is typechecking in there though for the publish